pkg/apis/config/v1alpha1/types.go

```diff
@@ -36,8 +36,9 @@ type ControllerConfiguration struct {
 	// ETCD is the etcd configuration.
 	ETCD ETCD `json:"etcd"`
 	// HealthCheckConfig is the config for the health check controller
-	// +optional
 	HealthCheckConfig *healthcheckconfigv1alpha1.HealthCheckConfig `json:"healthCheckConfig,omitempty"`
+	// BastionConfig the config for the Bastion
```

Comment:
Missing optional tag?

```suggestion
	// BastionConfig the config for the Bastion
	// +optional
```

Reply:
committed per comments 43a97228